setup.py

```diff
@@ -141,7 +140,6 @@ def run_setup(with_cext):
             'Intended Audience :: Developers',
             'Intended Audience :: System Administrators',
             'License :: OSI Approved :: Apache Software License',
-            'Programming Language :: Python :: 2.6',
```

Comment:
As you are updating this and added 3.7 to the Travis config, could you also add 3.6 and 3.7 here?

Reply:
Done.